libs/application-generic/src/usecases/workflow/update-workflow/update-workflow.usecase.ts

```diff
@@ -37,7 +37,7 @@ import {
   CreateMessageTemplateCommand,
   NotificationStep,
   NotificationStepVariantCommand,
-} from '../../../usecases';
+} from '../..';
```

Comment:
Same as above, risk of cyclic dependency
```suggestion
} from '../../usecases';
```